DED-592 Link sample apps to appropriate APIs/methods (#1358)
livechat/livechat-public-docs: src/pages/management/configuration-api/index.mdx

</Section>

> ## Sample App

Reviewer: How about we move this section below Methods? So that "Methods" is always the first section in each category

Author: Yup, I agree. Looks better when I check on the localhost😊

> ## Sample App

Reviewer: ```suggestion
## Sample app
```

> ### Tag Master

Reviewer: ```suggestion
```

Reviewer: I think we can remove this headline as there's only one sample app

<Text>

## Sample app

> 

Reviewer: How about mentioning the auth info here as well?

> Tag Master allows you to create, view, and delete tags and canned responses in an easy and predictable way. The app uses **Configuration API v3.5** endpoints:

Author: ```suggestion
Tag Master allows you to create, view, and delete tags and canned responses in an easy and predictable way. The app uses <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat/#accounts-sdk" target="_blank">Accounts SDK</a> and its <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat#redirect-1" target="_blank">Redirect</a> method for authorization and **Configuration API v3.5** endpoints:
```

> Tag Master allows you to create, view, and delete tags and canned responses in an easy and predictable way. The app uses **Configuration API v3.5** endpoints:

Author: ```suggestion
Tag Master allows you to create, view, and delete tags and canned responses in an easy and predictable way. It uses  <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat/#accounts-sdk" target="_blank">Accounts SDK</a> and its <a href="https://developers.livechat.com/docs/authorization/sign-in-with-livechat#redirect-1" target="_blank">Redirect</a> method for authorization and  **Configuration API v3.5** endpoints:
```
